pages/common/badblocks.md

```diff
@@ -0,0 +1,15 @@
+# badblocks
+
+> Search a device for bad blocks.
+
+- Search a disk for bad blocks by using a non-destructive read-only test:
+
+`sudo badblocks {{/dev/sda}}`
+
+- Search an unmounted disk for bad blocks with a non-descructive read-write test:
```

Comment:
descructive => typo again.

Are you deliberately trying to sneak in typos just to see whether we indeed review things or not ? :stuck_out_tongue_winking_eye: